### PR TITLE
Sparse unary ops

### PR DESCRIPTION
**Pull request by VinodSKumar**
Sparse unary ops (ceil, floor, round, trunc)

### CODE REVIEW

**Comment by ezyang** [comment]
@pearu, @aocsa, or @v0dro, any of you three want to do review on this?

**Comment by aocsa** [comment]
> @pearu, @aocsa, or @v0dro, any of you three want to do review on this?

Sure,  I can do that. Let me check.

**Comment by aocsa** [comment]
Thanks, @VinodSKumar,  for bringing here (`ceil, floor, round, trunc`) unary operators for sparse tensors. I did few comments related to c++ code. Moreover, I did some recommendation to the tests.  I referenced some tests from numpy and c++ because I think that we should use operation related inputs, specially for these operators.

**Review by aocsa — COMMENTED** [review]
Thanks, @VinodSKumar, for bringing here (ceil, floor, round, trunc) unary operators for sparse tensors. I did few comments related to c++ code. Moreover, I did some recommendation to the tests. I referenced some tests from numpy and c++ because I think that we should use operation related inputs, specially for these operators.

File: test/test_sparse.py [hunk 1]
```diff
@@ -1948,6 +1948,244 @@ def test_neg_negative(self):
...
+
+                # test on empty sparse tensor
+                input_uncoalesced = torch.sparse_coo_tensor(
+                    indices=torch.zeros([2, 0]),
+                    values=torch.zeros([0, 5, 5, 5, 5, 5, 5, 0]),
+                    size=[0, 0, 5, 5, 5, 5, 5, 5, 0],
+                    device=device,
+                    dtype=data_type
+                )
+                self._test_round(input_uncoalesced)
```
[the remark this answers is not in the document]
Comment (VinodSKumar) [reply test/test_sparse.py]: Thank you for the comment, updated the values to test functionality of the ops

**Pull request closed without merging**